blog/coal_seismoacoustic/coal_seismoacoustic.qmd

```diff
@@ -0,0 +1,136 @@
+
+![
+Longwall Mining concept showing A) An oblique view of the longwall operating, B) A close up
+of the shearer, and C) a profile view of the overburden. After @karacan2008modeling.
+](images/longwall_concept.png){#fig-longwall width="90%"}
+
+
+# Array Development and Deployment
+
+The core technology of the new monitoring approach is a type of distributed fiber optic sensing (DFOS)
+known as distributed acoustic sensing (DAS). DAS uses low-power, rapid laser pulses to measure subtle
+strains in standard fiber-optic cable. DAS has been used extensively in the oil and gas industry, and
+for other geophysical purposes [@lindsey2021fiber] but is only starting to find applications in mining.
+
+Our array consists of lead-cables, which are attached to the shields of the
+longwall, and microphones, or "acoustic transducers", which are just fiber optic
+cable wrapped around thin-walled plastic cylinders.
+
+![
+DAS seismoacoustic array. The array consists of lead cables (red) and microphones (yellow)
+](images/seismo_array.png){#fig-array width="90%"}
+
+We deployed the array on the longwall of an underground mine in the United States.
+The lead cables (red) were zip-tied to the shields and 14 microphones were deployed
+across the face.
+
+![Array deployment](images/deployment.png){#fig-array-deployment width="60%"}
+
+# Data
+
+During the deployment, several seismic events were detected by a surface seismic network.
+
+Here are a few examples of the audio recorded by the DAS microphones during these events:
+
+<figure>
+<figcaption>event 1</figcaption>
+<audio controls src="https://docs.google.com/uc?export=download&id=151wOqhF8q7TYQHrrxoohPpnAh75NPWrM"> type='audio/wav'
+        Your browser does not support the<code>audio</code> element.
+</audio>
+</figure>
+
+<figure>
+<figcaption>event 2</figcaption>
+<audio controls src="https://docs.google.com/uc?export=download&id=1LBwFSCAcQW8Ay5Cbjlr2JG-XaPlVPABE"> type='audio/wav'
+        Your browser does not support the <code>audio</code> element.
+</audio>
+</figure>
+
+Examining the recorded waveforms, distinct characteristics of lead cables, which again, are attached
+to the longwall shields, and the acoustic streams recorded by the microphones are apparent:
+
+![Data recorded by the seismoacoustic array
+](images/waveforms.png)
+
+The lead cables (blue) tend to record more impulsive, shorter duration signals across the entire
+array whereas the microphones record emergent, long duration signals on a few channels
+close to the event damage.
+
+# Monitoring Implications
+
+The array overcomes many of the shortcomings of traditional in-mine seismic monitoring for
+coal mines: sensors seldom need to be moved, and no permissible fiber-optic cable is used
+for the sensing elements.
+
+Additionally, the array records near-field data (sounds and vibrations) of bursting damage.
+These types of recordings may provide information about the extent and mechanisms of the damage,
```

Comment:
"may provide more information"? Or even "more informative information"? Can we make it even more redundant? Informative is clearly not quite the right word choice. "fundamental"? "revealing"? "gossipy"? (I'm done now. I blame thresaurus.com for that last one.)